src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebug/BlazorDebugConfigurationProvider.ts

```diff
@@ -23,9 +23,18 @@ export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurati
             await this.launchApp(folder, configuration);
         }
 
-        this.vscodeType.debug.onDidStartDebugSession(async event => {

```

Comment:
We were waiting for the application server to launch before spinning up the browser since the browser used to have a dependency on the debugging proxy was launched from the app server.

Now that we launch the debug proxy independently, we don't need to wait for the app server to start.